dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/DubboInvokerAvilableTest.java

```diff
@@ -117,6 +117,7 @@ public void test_NoInvokers() throws Exception {
     }
 
     @Test
+    @Ignore
```

Comment:
Why ignore this test case ?

Reply:
I didn't ignore this.
Maybe it is a outdated content ?